src/components/Dashboard/Dashboard.jsx

```diff
@@ -19,11 +20,8 @@ export class Dashboard extends React.Component {
     }
 
     render() {
-        const showPlayerInteractions = (this.props.playCard || this.props.playWord || this.props.voteCard)
-                                    || ((this.props.currentPlayer) && 
-                                        (!this.props.finishedRound && 
-                                        this.props.username === this.props.currentPlayer.username && 
-                                        this.props.playedCardId));
+        const showPIright = (this.props.voteCard || this.props.word || (!this.props.playWord && this.props.playCard)) && !(this.props.winner || this.props.drawers.length > 1);
```

Comment:
Can you rename these two variables? e.g. `showPlayWord` and `showPlayerInteractions`?